app/assets/stylesheets/organizations.scss

```diff
@@ -1,3 +1,21 @@
 // Place all the styles related to the Organizations controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: https://sass-lang.com/
+$form-padding: 40px;
+$form-margin: 20px 0;
+$field-padding: 10px 0;
```

Comment:
Hey @ncbui! Wanted to say, maybe it is better to have a _variables.scss file where we add all the variables? So in that way they can be used across all the sass files, as those padding and margins will be probably used in other views as well?

Reply:
Hey @saradotlog , that is a good point for variables like this. Let's discuss it at the stand up. For this PR specifically, this suggestion is beyond it's scope. I have not taken the time to test this out on all other resources and cannot presume to say that these specific margins and padding will work for everyone.

Reply:
Yeah, it is, no point in solving it here. Just saying that maybe we should add that variables file before we all start adding variables that might do the same. It is luckily that the edit forms have the same kind of styling, for orgs, workers and even shifts, so the styles might be shared. It's true that we might need to do a global overview

Reply:
Thanks for reviewing this @saradotlog, I'm going to refactor this to use more of bootstrap's utility classes.